### PR TITLE
#109 [Component][Authentication] Reload automatically browser's window a...

### PR DESCRIPTION
**Pull request by fkroockmann**
Currently we have to do it manually which is not convenient. Of course it should be down in the only case that authentication succeed.

### CODE REVIEW

File: src/tb/component/authentication/main.js [hunk 1]
```diff
@@ -69,16 +68,22 @@ define(
              * @param {String} password
              */
             authenticate: function (username, password) {
-                var self = this;
+                var self = this,
+                    onDone = function (data, response) {
+                        console.log('la');
```
Comment (mickaelandrieu) [inline src/tb/component/authentication/main.js]: wtf.

File: src/tb/component/authentication/main.js [hunk 2]
```diff
@@ -127,7 +132,7 @@ define(
              * @param {Object} response
              */
             onRequestDone: function (response) {
-
+                console.log(response);
```
Comment (mickaelandrieu) [inline src/tb/component/authentication/main.js]: why do you need to log the response ? use logger if you need to keep a trace ;)

**Comment by mickaelandrieu** [comment]
@fkroockmann is hot today !

**Comment by ndufreche** [comment]
:+1:

**Pull request merged**